ale_linters/go/gotype.vim

```diff
@@ -0,0 +1,10 @@
+" Author: Jelte Fennema <github-public@jeltef.nl>
+" Description: gotype for Go files
+
+call ale#linter#Define('go', {
+\   'name': 'gotype',
+\   'output_stream': 'stderr',
+\   'executable': 'gotype',
+\   'command': 'gotype -e %t $(find $(dirname %s) -maxdepth 1 -name "*.go" | grep -v "^"%s"$")',
```

Comment:
This command uses Bash syntax. This isn't portable.

Does this command accept files via stdin? Or do you need to pass all of the `.go` files to the command? ALE adds `< TEMPORARY_FILENAME` by default to the end of the command with the contents of the buffer.